src/styles/footer.scss

```diff
@@ -0,0 +1,29 @@
+h3 {
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+#footer {
+  align-items: center;
+  bottom: 1.3vw;
+  display: flex;
+  font-family: 'Westwood Sans','Helvetica Neue', 'Helvetica', sans-serif;
+  height: 1.35vw;

```

Comment:
Can you take out the height and width? It's not necessary. Also, this branch isn't updated with the mobile design so I don't know how the footer will look on mobile. But just by looking at the code, it should be okay. If it's not I'll fix it in my issue for the week.